server/partials/footer.html

```diff
@@ -22,9 +22,11 @@ <h4 class="ui header">Community</h4>
         <h4 class="ui header">Social</h4>
         <div class="ui link list">
           <a class="item" href="https://twitter.com/fomanticui" target="_blank">
-            <i class="twitter icon"></i>
+            <svg width="1200" height="1227" viewBox="0 0 1200 1227" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
should the icon be updated in FUI assets instead of manually inserting the new svg here?

Reply:
Check the conversion above what we have discussed

Reply:
Yes. We will later upgrade to FontAwesome 6.x which has the new X brand icon. See the [comment](https://github.com/fomantic/Fomantic-UI-Docs/pull/479#issuecomment-1745510759) above.